_run/common-kind.mk

```diff
@@ -1,19 +1,41 @@
-KIND_NAME      ?= $(shell basename $$PWD)
+KIND_NAME      = kind
```

Comment:
can you set `KIND_NAME=kind` in the environment in the workflow yaml?

Reply:
There isn't a strait `name` param in the config :shrug: Kinda odd but seems to be intentionally a CLI flag.

Reply:
I mean [set an environment variable](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env).  It is using `?=`; the right-hand side here is a default if `$KIND_NAME` isn't defined.

Reply:
Done!

Reply:
I want to keep the previous behavior - it defaults to the name of the current directory.

Reply:
Ahh, I see. Reset it.

The reason I changed it was there was a problem with the cluster being named `kube-control-plane` and messing up downstream queries. However Looks like the variable is getting applied where applicable so local testing shows it's working with $$PWD!

Reply:
You can override the make variable's default value with an environment variable.  This is what I've been talking about the whole time.

I don't know why the cluster name would be `kube-control-plane`, hopefully that was just a fluke.

Reply:
looks like it's still a problem @Ropes 

```Error: No such container: kube-control-plane
Error: No such container: kube-control-plane
# ASSUMES:
# 1. cluster created - `kind create cluster --config=_run/kube/kind-config.yaml`
# 2. cluster setup - `make -s -C _run/kube kind-ingress-setup`
cp akashctl akashd ./_build
KIND_APP_PORT="" KIND_K8S_IP="" go test -mod=readonly -p 4 -tags "e2e integration " -v ./integration/... -run TestE2EApp
```

Reply:
The naming that `engineerd/setup-kind` uses is rather peculiar. eg the actual k8s cluster name is `kind-kind` by default, and docker container name is `kind-control-plane`(locally `kube-control-plane`).
```
 Kube Environment0s
kube-system     service/metrics-server                       ClusterIP   10.96.40.80    <none>        443/TCP                      67s
Run kubectl config view
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: DATA+OMITTED
    server: https://127.0.0.1:32768
  name: kind-kind
contexts:
- context:
    cluster: kind-kind
    user: kind-kind
  name: kind-kind
current-context: kind-kind
```

Tests are now passing again by setting the job's env KIND_NAME to tell the Makefiles where to look.